### DISCUSSION TITLE
Running XCT.Samples in either UWP and WPF - drilling into most items shows blank page

### DISCUSSION THREAD

**Discussion by IronMikeyB** [xamarin/XamarinCommunityToolkit#1186] [Q&A] [answered]
I've tried running the XCT.Sample solution on both UWP and WPF.  It runs fine displaying the intro page with various functionality sections (Behaviors, C# Markup, Converters, etc).  

Selecting any of these (except C# Markup) results in a blank page.  

Something I might be missing here?  Running under Android works fine for me.

**Comment by Cfun1** [comment, marked as answer]
Hi, it is a known issue https://github.com/xamarin/XamarinCommunityToolkit/issues/606

**Reply by IronMikeyB** [reply]
Thanks for the speedy reply
